### ISSUE TITLE
Cannot add T411 in Sonarr

### ISSUE THREAD

**Issue by alex3137** [zone117x/Jackett#218]
Hello.

I am able to add T411 as an indexer in Jacket but when I try to add it in Sonar, I get the following error:

No results were returned from your indexer, please check your settings.

I am using Jackett Version 0.6.8.0 and Sonarr Ver. 2.0.0.3357. Other indexers are fine.

Thanks

**Comment by raspdealer** [comment]
Can you post your sonarr settings ?

**Comment by alex3137** [comment]
See a screenshot of the indexer setting in Sonarr:

![Sonar rerror](https://cloud.githubusercontent.com/assets/6026026/10428984/0ecba662-70ed-11e5-8d68-24c9dc30c968.jpg)

**Comment by raspdealer** [comment]
Have you mapped categories in jackett ?

**Comment by alex3137** [comment]
I believe so:

![jacket](https://cloud.githubusercontent.com/assets/6026026/10430559/c5cddaba-70f7-11e5-8f03-c030850dab71.jpg)

**Comment by raspdealer** [comment]
You need to add a custom Torznab in sonarr not a preset one.

**Comment by alex3137** [comment]
Oh right but now I am getting a different error:

![sonarr](https://cloud.githubusercontent.com/assets/6026026/10437271/d1ba47d8-7123-11e5-9919-66c74d355964.jpg)

In Sonarr logs, I have HTTP 500 errors:

`HTTP Error - Res: [GET] http://10.10.10.119:9117/torznab/t411/api?t=tvsearch&cat=5030,5040&extended=1&apikey=<removed>&offset=0&limit=100 : 500.InternalServerError`

`Unable to connect to indexer: HTTP request failed: [500:InternalServerError] [GET] at [http://10.10.10.119:9117/torznab/t411/api?t=tvsearch&cat=5030,5040&extended=1&apikey=<removed>&offset=0&limit=100]: HTTP request failed: [500:InternalServerError] [GET] at [http://10.10.10.119:9117/torznab/t411/api?t=tvsearch&cat=5030,5040&extended=1&apikey=ofo60okuu24x5j1x1p8o3cwvoedxwr5z&offset=0&limit=100]`



Jacket logs:

http://pastebin.com/5aM6aeL3#

**Comment by raspdealer** [comment]
Is your jackett online ?

**Comment by alex3137** [comment]
Yes and searches to other indexers (TPB, ShowRSS...) are working fine.

**Comment by spookssi** [comment]
Same error for me :)...

**Comment by Snoflex** [comment]
I have the same issue:

In Jackett, T411 and all other work fine, but in Sonarr, T411 doesn't work and others do.

Hopefully someone will know where this comes from.

**Comment by alex3137** [comment]
Anybody ?

**Comment by sdesbure** [comment]
Hello,
I've got the same issue. I'll try to jump into the code if I have time

**Comment by alex3137** [comment]
Would be great at it seems Jacket development is on hold. Dev added few days ago following metion in the readme file:

`This project is currently in hibernation and is unmaintained, feel free to fork and continue development.`

This is a great and useful project, hoping community can carry on its development.

**Comment by raspdealer** [comment]
You can open your issue on this fork https://github.com/Jackett/Jackett/
